util/logtools/encounter_tools.ts

```diff
@@ -144,58 +144,55 @@ export class EncounterFinder {
 
     // We are in a combat zone, so we next check for victory/defeat.
     // If either is found, end the current encounter.
-    const cW = this.regex.cactbotWipe.exec(line);
+    // However, *if* seals are present, we want to use them to end the current encounter,
+    // rather than using combat status.
```

Comment:
> I noted it in-line, but if we don't explicitly check for seals when ending combat, stuff gets really weird. If there are seals present in the current zone, we want to use only seals for starting/stopping combat, since those are the only encounters we care about

I'm not sure I understand why.  What's wrong with ending with either combat or seals?

Reply:
"Really weird" might be an overstatement, (that's mostly from my memory of automating `make_timeline` in the first place,) but `split_log` throws an error if we don't handle seals this way. I would have to dig through a little to see exactly why,  but interleaving starts and stops this way can leave us with encounters whose start or end times aren't defined.